tests/image-stream/utils.spec.js

```diff
@@ -26,44 +26,34 @@ describe('ImageStream: Utils', function() {
 
   describe('.getArchiveMimeType()', function() {
 
-    it('should resolve application/x-bzip2 for a bz2 archive', function(done) {
+    it('should resolve application/x-bzip2 for a bz2 archive', function() {
```

Comment:
> mocha knows how to handle promises

Wow, I had no idea this was the case, and I've been using `done` all this time. If you can correct those, that'd be awesome :+1: